hicommonwealth/commonwealth
# Clicking create thread on a contest topic doesn't populate the desired topic at first.

## Link to Issue
Closes: #10029

## Description of Changes
- Updated the default topic logic in useNewThreadForm to ensure the thread topic state updates.


## "How We Fixed It"

## Test Plan
- Navigate to the discussion Page.
- Select the Contest Topic.
- Click "Create thread".
- Verify that the topic is selected.
- Also repeat the above steps with the FF "newEditor" turned on for the new Editor.

## Deployment Plan

1. 

## Other Considerations

-
